packages/create-llama/package.json

```diff
@@ -24,7 +24,7 @@
     "dev": "ncc build ./index.ts -w -o dist/",
     "build": "npm run clean && ncc build ./index.ts -o ./dist/ --minify --no-cache --no-source-map-register",
     "lint": "eslint . --ignore-pattern dist",
-    "e2e": "playwright test",
+    "e2e": "pnpm pack --pack-destination ./output && npm i -g ./output/create-llama-*.tgz && playwright test",
```

Comment:
1. clean up  `output` in `clean` target and it to gitignore
2. make build target `build:package` with `pnpm build && pnpm pack --pack-destination ./output`
3. use `build:package` in `e2e` target